### ISSUE TITLE
StepperPanel: Pass Through "number" option not working

### ISSUE THREAD

**Issue by benjaminarteaga** [primefaces/primereact#6910]
### Describe the bug

The `number` option within the `pt` property of `StepperPanel` is not applying the specified class. 

### Reproducer

https://stackblitz.com/edit/bglgem?file=src%2FApp.jsx

### PrimeReact version

10.7.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Vite

**Comment by melloware** [comment]
Is your issue similar to this one: https://github.com/primefaces/primereact/issues/6870 ?

**Comment by benjaminarteaga** [comment]
I don't think so, it is similar to [#5932](https://github.com/primefaces/primereact/issues/5932).
I tested both with unstyled mode and with a theme, and the classes I added via `pt` are simply not being assigned in the DOM. This happens with `number` and some other `pt` properties in `StepperPanel`.
